assets/scss/components/_button.scss

```diff
@@ -257,6 +258,17 @@ Class                   | Description
 	background: $C_contentBG;
 }
 
+.button--reset {
+	@include buttonHover(transparent);
+	background: transparent;
```

Comment:
filed a separate ticket for specificity issue:
https://meetup.atlassian.net/browse/WC-42